importer/importer_utils.py

```diff
@@ -626,4 +626,21 @@ def create_site_instance(language, family):
         site = pywikibot.Site(language, family)
         site_cache[site_key] = site
     return site
-    
\ No newline at end of file
+
+
+def package_quantity(value, unit=None):
+    """Package a quantity value in a standardised form."""
+    quantity = {"quantity_value": value}
+    if unit:
+        quantity["unit"] = unit
+    return quantity
+
+
+def package_time(text, date_dict):
```

Comment:
The method doesn't make use of the `text` parameter.

Reply:
good catch